wrapper/src/main/java/software/amazon/jdbc/plugin/readwritesplitting/ReadWriteSplittingPlugin.java

```diff
@@ -214,6 +218,7 @@ public <T, E extends Exception> T execute(
         LOGGER.finer(
             () -> Messages.get("ReadWriteSplittingPlugin.failoverExceptionWhileExecutingCommand",
                 new Object[] {methodName}));
+        ConnectionProviderManager.releaseResources();
```

Comment:
It doesn't look like a responsibility of this plugin. Need to discuss.

Reply:
Is there any interference here with connection tracker plugin?

Reply:
Would it make more sense to move it to the failover plugin then?
Can you remind me what the connection tracker does and how this might interfere with it?

Reply:
If you move it to failover plugin, I still can't understand how to gently release certain connection pools. There's no designed communication between these two component. Calling ConnectionProviderManager.releaseResources() will effectively close all connection pools including those that contains readers connections that haven't been affected by failover.

Reply:
Let's discuss how/where to call releaseResources. My testing shows that all connections that existed before failover are broken after failover (regardless of any instance URL or cluster URL used) so I think all connection pools will need to be closed.